frontend/src/bundles/users/store/slice.ts

```diff
@@ -31,6 +32,16 @@ const { reducer, actions, name } = createSlice({
         builder.addCase(loadAll.rejected, (state) => {
             state.dataStatus = DataStatus.REJECTED;
         });
+        builder.addCase(signUp.pending, (state) => {
+            state.dataStatus = DataStatus.PENDING;
+        });
+        builder.addCase(signUp.fulfilled, (state, action) => {
+            state.users = [...state.users, action.payload.user];
+            state.dataStatus = DataStatus.FULFILLED;
```

Comment:
This slice for fetching users from separate api endpoint, we don't need auth actions cases here

Reply:
all this logic related to the auth store only!